src/back/auth/src/repo/table_session.cpp

```diff
@@ -0,0 +1,89 @@
+#include "table_session.hpp"
+
+#include "userver/components/component_config.hpp"
+#include "userver/components/component_context.hpp"
+#include <userver/yaml_config/merge_schemas.hpp>
+#include <userver/storages/postgres/component.hpp>
+
+namespace svetit::auth::table {
+
+Session::Session(
+	    storages::postgres::ClusterPtr pg
+        )
```

Comment:
Если у класса один аргумент в конструкторе или если они короткие, то можно без лишних переносов строк.
```suggestion
Session::Session(storages::postgres::ClusterPtr pg)
```

Reply:
поправил везде